src/OrchardCore.Modules/OrchardCore.Github/README.md

```diff
@@ -0,0 +1,24 @@
+# Github (OrchardCore.Github)
```

Comment:
Missing the entry in the main documentation index?

Reply:
ping

Reply:
Didn't know how to do it. Found it in mkdocs.yml